cmd/csi-driver/Dockerfile

```diff
@@ -33,4 +33,7 @@ ADD [ "csi-driver", "/bin/" ]
 ADD [ "conform/entrypoint.sh", "/entrypoint.sh" ]
 RUN [ "chmod", "+x", "/entrypoint.sh" ]
 
+# add path to hpe scripts
+ENV PATH="/usr_local/local/bin:${PATH}"
```

Comment:
since we are using kubectl to exec and collect logs we don't need to put the script on host path. It can be within container /usr/local/bin etc and not hosts user_local path.

Reply:
since we have mapped /usr/local to /usr_local/local inside the container the scripts are not in path. I get the following error without setting this path

root@rkumar-k8s-cl1-1:~# ./hpe-kubectl-diagnostic.sh rkumar-k8s-cl1-2
OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"hpe-logcollector.sh\": executable file not found in $PATH": unknown
command terminated with exit code 126

Reply:
Yes, if we change cp from /usr_local to container path /usr/local then it will be part of PATH variable.

Reply:
yup works with the change. Thanks